### PR TITLE
Jschiller/tryout uncommitted

### PR DESCRIPTION
**Pull request by jasonbs10**
Updated scootapi to handle new scheduler strategy parameters. TODO: scheduler has a killjob() panic.

### CODE REVIEW

File: scootapi/scoot.thrift [hunk 1]
```diff
@@ -55,7 +55,7 @@ struct TaskDefinition {
 
 struct JobDefinition {
   1: required list<TaskDefinition> tasks,
-  2: optional JobType jobType,
+  2: optional JobType DEPRECATED_jobType,
```
Comment (JeanetteBruno) [inline scootapi/scoot.thrift]: will we be removing this? (I'm assuming we're leaving it in to limit the scope of this change)

Comment (jasonbs10) [inline scootapi/scoot.thrift]: We can remove this after everything has been redeployed (but leave something to indicate that field '2:' is no longer available.)

File: scootapi/server/run_job.go [hunk 2]
```diff
@@ -65,8 +65,17 @@ func thriftJobToScoot(def *scoot.JobDefinition) (result sched.JobDefinition, err
...
+	}
+	if def.Basis != nil {
+		result.Basis = *def.Basis
+	}
 	if def.JobType != nil {
-		result.JobType = def.JobType.String()
+		result.JobType = *def.JobType
+	}
+	if def.Priority != nil {
+		result.Priority = sched.Priority(*def.Priority)
```
Comment (dgassaway) [inline scootapi/server/run_job.go]: Do we need one for requestor?

Comment (jasonbs10) [inline scootapi/server/run_job.go]: Yep, thanks.

**Pull request merged**